app/src/components/guild/index.tsx

```diff
@@ -0,0 +1,276 @@
+import { BigNumber } from 'ethers/utils'
+import React, { useEffect, useState } from 'react'
+import styled from 'styled-components'
+
```

Comment:
Great that you created guild in the components folder I did the same thing on mine! Also In order to keep separation of concerns I think it might make sense to create guild folder inside pages folder and store all the logic there and separate it from view which will be stored in components? Curios what you and Kaden think about this and how we should go about it...You can also checkout maybe how I did it and if it makes sense or if we should do It differently.... @KadenZipfel @hexyls

Reply:
I really like this idea, made a `proposed_rewards_page.tsx` and a `proposed_rewards_view.tsx`, much better now